compass/landice/tests/mismipplus/namelist.landice

```diff
@@ -0,0 +1,27 @@
+config_start_time = '0000-01-01_00:00:00'
+config_run_duration = '0005-00-00_00:00:00'
+config_adaptive_timestep = .true.
+config_adaptive_timestep_CFL_fraction = 0.9
```

Comment:
Why is this different from the CFL fraction of 0.95 in https://github.com/MPAS-Dev/compass/pull/386/files#diff-acd02e538bc2c87b44f42baa553a3630dd419f20c83568eac48f8b2a9fc497b7 ?

Reply:
(that link is to the namelist.landice file in dt_convergence_test)